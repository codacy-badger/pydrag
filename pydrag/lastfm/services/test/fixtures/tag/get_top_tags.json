{
    "version": 1,
    "interactions": [
        {
            "request": {
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&format=json&method=tag.getTopTags&num_res=10&offset=20",
                "body": null,
                "headers": {
                    "User-Agent": [
                        "python-requests/2.20.1"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Accept": [
                        "*/*"
                    ],
                    "Connection": [
                        "keep-alive"
                    ]
                }
            },
            "response": {
                "status": {
                    "code": 200,
                    "message": "OK"
                },
                "headers": {
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ],
                    "Content-Length": [
                        "568"
                    ],
                    "Access-Control-Allow-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "TSVC-Time-Spent": [
                        "0.000"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS, PUT"
                    ],
                    "Date": [
                        "Sun, 11 Nov 2018 15:56:16 GMT"
                    ],
                    "Connection": [
                        "keep-alive"
                    ]
                },
                "body": {
                    "string": "{\"toptags\":{\"@attr\":{\"offset\":20,\"num_res\":10,\"total\":2693772},\"tag\":[{\"name\":\"black metal\",\"count\":808560,\"reach\":60919},{\"name\":\"dance\",\"count\":789981,\"reach\":131041},{\"name\":\"80s\",\"count\":768919,\"reach\":99629},{\"name\":\"Progressive rock\",\"count\":705995,\"reach\":94830},{\"name\":\"death metal\",\"count\":696989,\"reach\":70446},{\"name\":\"hardcore\",\"count\":679448,\"reach\":96344},{\"name\":\"british\",\"count\":679004,\"reach\":92377},{\"name\":\"heavy metal\",\"count\":676538,\"reach\":89132},{\"name\":\"soul\",\"count\":659170,\"reach\":99192},{\"name\":\"chillout\",\"count\":642670,\"reach\":102845}]}}"
                }
            }
        }
    ]
}
