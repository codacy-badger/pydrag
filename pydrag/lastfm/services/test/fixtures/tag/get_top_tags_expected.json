{
    "tag": [
        {
            "name": "black metal",
            "reach": 60919,
            "count": 808560
        },
        {
            "name": "dance",
            "reach": 131041,
            "count": 789981
        },
        {
            "name": "80s",
            "reach": 99629,
            "count": 768919
        },
        {
            "name": "Progressive rock",
            "reach": 94830,
            "count": 705995
        },
        {
            "name": "death metal",
            "reach": 70446,
            "count": 696989
        },
        {
            "name": "hardcore",
            "reach": 96344,
            "count": 679448
        },
        {
            "name": "british",
            "reach": 92377,
            "count": 679004
        },
        {
            "name": "heavy metal",
            "reach": 89132,
            "count": 676538
        },
        {
            "name": "soul",
            "reach": 99192,
            "count": 659170
        },
        {
            "name": "chillout",
            "reach": 102845,
            "count": 642670
        }
    ],
    "@attr": {
        "total": 2693772,
        "offset": 20,
        "num_res": 10
    }
}
